main.tf

```diff
@@ -1,21 +1,40 @@
-data "aws_caller_identity" "default" {}
+locals {
+  e = module.this.enabled # Shorthand for quick reference
```

Comment:
We call this `enabled` in all cases. Let's update that 👍  

```suggestion
  enabled = module.this.enabled
```

Reply:
👍🏼  Updated